### ISSUE TITLE
Projeto em avaliação

### ISSUE THREAD

**Issue by orlandosaraivajr** [brunounky/Hack-Burger-DSM-Fatec-Araras#1]
![image](https://github.com/brunounky/hack_burger_dsm_fatec_araras/assets/2480421/c7d04b20-fa64-41a4-bac8-84316dc3c45c)
O código foi feito sozinho ?
Feito em C++, poderia ter separado cada trecho em uma função ou mesmo criado uma struct para o lanche.
Há pontos de melhorias no código encontrado.

Parabéns pelo trabalho

**Comment by brunounky** [comment]
Sim, eu atrasei a entrega do trabalho e o João deuxou eu entregar individualmente.
 Fiz em C++ pois era dava pra fazer de forma mais rapida, pelo menos tenho mais familiaridade. Vou pensar em fazer uma struct da próxima vez, e funções como calcular custo e valor de venda separadamente, usei os parâmetros ja passados pelo Joao, mas se desse para o usuario escolher a porcentagem em cima do custo que ele quer seria melhor.
